.github/workflows/continuous_deployment.yml

```
@@ -0,0 +1,35 @@
+    workflows: ["Continuous Delivery"] # Runs after images are created properly
+    branches: [dev]
+    types:
+      - completed
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)